iocore/dns/DNS.cc

```diff
@@ -766,6 +766,8 @@ DNSHandler::failover()
     ip_text_buffer buff;
     Warning("failover: connection to DNS server %s lost, retrying", ats_ip_ntop(&ip.sa, buff, sizeof(buff)));
   }
+  // Make sure retries are done even if no more requests.
+  this_ethread()->schedule_at(this, DNS_PRIMARY_RETRY_PERIOD);
```

Comment:
Shouldn't it be `schedule_in`?

Reply:
Interesting question. The code is cloned from [here](https://github.com/apache/trafficserver/blob/master/iocore/dns/DNS.cc#L1021) in `DNSHandler::mainEvent`. Maybe that's broken too.

Reply:
Yeh, that looks wrong, examining the code, _at vs _in is clear. If you want to use _at, it would need to be

```
this_ethread()->schedule_at(this, Thread::get_hrtime() + DNS_PRIMARY_RETRY_PERIOD);
```

Reply:
OK, I'll fix both. Still struggling with other weird things going on.